src/lib/select/select.ts

```diff
@@ -242,7 +244,8 @@ export class MdSelect implements AfterContentInit, ControlValueAccessor, OnDestr
   ngAfterContentInit() {
     this._initKeyManager();
     this._resetOptions();
```

Comment:
You should be able to remove this call now that it's happening again in line 249